feature-libs/organization/administration/styles/_page-template.scss

```diff
@@ -0,0 +1,17 @@
+@import 'company-page-template';
+
+// The blacklist can be used to prevent template CSS
+// being generated.
+$page-template-blacklist-myco: () !default;
```

Comment:
good stuff. Only can we make this 
`$page-template-blacklist-organization` and `$page-template-whitelist-organization`. We try to avoid abbreviations.

Reply:
Done!